core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts

```diff
@@ -62,24 +66,24 @@ export class TranslateTextService {
   STARTING_INDEX = -1;
   PENDING = 'pending';
   SUBMITTED = 'submitted';
-  stateWiseContents = {};
-  stateWiseContentIds = {};
-  stateNamesList = [];
-  stateAndContent = [];
+  stateWiseContents: StateNamesToContentIdMapping = {};
+  stateWiseContentIds: Record<string, string[]> = {};
+  stateNamesList: string[] = [];
+  stateAndContent: StateAndContent[] = [];
   activeIndex = this.STARTING_INDEX;
-  activeExpId;
-  activeExpVersion;
-  activeContentId;
-  activeStateName: string;
-  activeContentText: string;
-  activeContentStatus: Status;
+  activeExpId!: string;
+  activeExpVersion!: string;
+  activeContentId!: string;
+  activeStateName!: string;
+  activeContentText?: string | string[];
+  activeContentStatus?: Status;
```

Comment:
```typescript
  // These properties are initialized using Angular lifecycle hooks
  // and we need to do non-null assertion, for more information see
  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
  activeExpId!: string;
  activeExpVersion!: string;
  activeContentId!: string;
  activeStateName!: string;
  activeContentText!: string | string[];
  activeContentStatus!: Status;
  STARTING_INDEX = -1;
  PENDING: Status = 'pending';
  SUBMITTED: Status = 'submitted';
  stateWiseContents: StateNamesToContentIdMapping = {};
  stateWiseContentIds: Record<string, string[]> = {};
  stateNamesList: string[] = [];
  stateAndContent: StateAndContent[] = [];
  activeIndex = this.STARTING_INDEX;
```
You will see a similar pattern in some of other strictly typed files as well. Note here that two properies `activeContentText` and `activeContentStatus` need not to be made optional.